test/driver_sanity/kernel_compat/scripts/vagrant_scap_open_test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+if [[ $# -ne 1 || "${EUID}" -ne 0 ]]; then
+  echo "Usage: sudo bash vagrant_scap_open_test.sh COMPILER"
+  echo "Run as root user in VM"
+  exit 1
+fi
+
+COMPILER="${1}";
+set -eou pipefail
+
+echo "Test run for compiler/kernel ${COMPILER}/$(uname -r)"
+
+rm -f /tmp/o;
```

Comment:
Looks like we might not care about preserving this file, if we are rebooting the machine between calls to this script, why not use a temporary file instead?
```suggestion
OUTPUT_FILE="$(mktemp)"
```